### PR TITLE
Expose the docker container ports to the jenkins server on powerci

### PR DESCRIPTION
**Pull request by ramereth**

### CODE REVIEW

**Comment by ramereth** [comment]
[POWER CI: Update hardcoded paths to work with current infra](https://trello.com/c/e0BctJxL/541-power-ci-update-hardcoded-paths-to-work-with-current-infra)

File: spec/unit/recipes/powerci_spec.rb [hunk 1]
```diff
@@ -28,6 +28,12 @@
             ]
           )
       end
+      it do
+        expect(chef_run).to create_iptables_ng_rule('docker_expose_ipv4')
+          .with(
+            rule: ['--protocol tcp --source 192.168.6.0/24 --destination-port 32768:61000 --jump ACCEPT']
```
Comment (DesertBeagle) [inline spec/unit/recipes/powerci_spec.rb]: So these values are created when the `firewall` recipe runs?

**Comment by ramereth** [comment]
!bump patch *

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `master` and has automatically performed a patch-level version bump to v1.2.2.  Have a nice day!

**Pull request merged**